content/en/platform/corda/5.2/developing-applications/api/application/persistence.md

```diff
@@ -145,11 +147,13 @@ To use the Persistence API from a {{< tooltip >}}flow{{< /tooltip >}}:
 2. To create a `Dog` entity that writes a row to the database, use the following code:
    ```kotlin
      val dog = Dog(dogId, "dog", Instant.now(), "none")
-     persistenceService.persist(dog)
+     persistenceService.persist("persist-dog", dog)
     ```
 
+    `persist-dog` is a unique (within the context of the flow) deterministic deduplication ID to enable request idempotence in failure scenarios. The ID must be deterministic at the time of the function call and can not be more than 128 characters.
```

Comment:
We should note that, as the ID must be deterministic, you should _not_ use a random UUID